x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -136,7 +143,18 @@ func GetCalcSetNetworkRegrets(args GetCalcSetNetworkRegretsArgs) error {
 		if err != nil {
 			return errorsmod.Wrapf(err, "Error setting inferer regret")
 		}
-		if !newParticipant {
+
+		shouldAddWorkerRegret, err := isExperiencedInferer(
+			args.Ctx,
+			args.K,
+			topic,
```

Comment:
Pass the `TopicId` and `AlphaRegret` instead of the whole topic

Reply:
fixed in next commit